### ISSUE TITLE
Config file description in readme is confusing and does not work

### ISSUE THREAD

**Issue by peteh** [wppconnect-team/wppconnect-server#1225]
## Description
The readme talks about a config.json, however the current version seems to expect a config.ts. 

I used my config.json and just renamed it, which makes the system boot, but it does not seem to load the correct configured values when using the docker container. 

Additionally the docker-compose is pointing to /config.ts instead of ./config.ts on the local drive. 

## Environment

- **Wppconnect-server version(s):** [e.g. 2.0.1]
- **Browser:** [e.g. Chrome 87, Chromium 85]
- **OS:** Ubuntu 23.04
- **Node version:** [e.g. Node 8, Node 14] whatever is in the docker
- **WhatsApp version:** [e.g. 2.2126.14]
- **MultiDevice (BETA):** yes/no
- **config.json:** [Your src/config.json file content]
```
{
  "secretKey": "THISISMYSECURETOKEN",
  "host": "http://localhost",
  "port": "21465",
  "deviceName": "Smrt",
  "poweredBy": "Me",
  "startAllSession": true,
  "tokenStoreType": "file",
  "maxListeners": 15,
  "customUserDataDir": "./userDataDir/",
  "webhook": {
    "url": "http://10.10.0.1:9000/incoming",
    "url2": "http://smrt:9000/incoming",
    "autoDownload": false,
    "uploadS3": false,
    "readMessage": true,
    "allUnreadOnStart": false,
    "listenAcks": true,
    "onPresenceChanged": false,
    "onParticipantsChanged": false,
    "onReactionMessage": true,
    "onPollResponse": false,
    "onRevokedMessage": false
  },
  "archive": {
    "enable": false,
    "waitTime": 10,
    "daysToArchive": 45
  },
  "log": {
    "level": "error",
    "logger": ["console", "file"]
  },
  "createOptions": {
    "browserArgs": [
      "--disable-web-security",
      "--no-sandbox",
      "--disable-web-security",
      "--aggressive-cache-discard",
      "--disable-cache",
      "--disable-application-cache",
      "--disable-offline-load-stale-cache",
      "--disk-cache-size=0",
      "--disable-background-networking",
      "--disable-default-apps",
      "--disable-extensions",
      "--disable-sync",
      "--disable-translate",
      "--hide-scrollbars",
      "--metrics-recording-only",
      "--mute-audio",
      "--no-first-run",
      "--safebrowsing-disable-auto-update",
      "--ignore-certificate-errors",
      "--ignore-ssl-errors",
      "--ignore-certificate-errors-spki-list"
    ]
  },
  "mapper": {
    "enable": false,
    "prefix": "tagone-"
  },
  "db": {
    "mongodbDatabase": "tokens",
    "mongodbCollection": "",
    "mongodbUser": "",
    "mongodbPassword": "",
    "mongodbHost": "",
    "mongoIsRemote": true,
    "mongoURLRemote": "",
    "mongodbPort": 27017,
    "redisHost": "localhost",
    "redisPort": 6379,
    "redisPassword": "",
    "redisDb": 0,
    "redisPrefix": "docker"
  }
}
```

## Steps to Reproduce

1. change docker-compose to read the config.ts from the local file instead of root ./config.ts

## Log Output
It's connecting as wppconnect instead of smrt to my whatsapp. So the entries from the config are not considered.

**Comment by cgslivre** [comment]
Same problema!

config.json:
```json
{
    "secretKey": "ASECRETKEY",
    ...
}
```

My solution:
1 - rename config.json to config.js
2 - wrap old json with "exports"
```javascript
"use strict";Object.defineProperty(exports, "__esModule", { value: true });
exports.default = void 0;
var _default = {
    "secretKey": "ASECRETKEY",
    ...
};
exports.default = _default;
```

The config.js is like this:
```javascript
"use strict";Object.defineProperty(exports, "__esModule", { value: true });exports.default = void 0;var _default = {
    "secretKey": "ASECRETKEY",
    ...
};exports.default = _default;
```

But this is not a solution! It's just a workaround to get around the problem!

**Comment by renat473** [comment]
fix commit 5ae0a20
